core/lib/spree/testing_support/common_rake.rb

```diff
@@ -24,7 +24,7 @@
     begin
       require "generators/#{ENV['LIB_NAME']}/install/install_generator"
       puts 'Running extension installation generator...'
-      "#{ENV['LIB_NAME'].camelize}::Generators::InstallGenerator".constantize.start(["--auto-run-migrations"])
+      "#{ENV['LIB_NAMESPACE'] || ENV['LIB_NAME'].camelize}::Generators::InstallGenerator".constantize.start(["--auto-run-migrations"])
```

Comment:
Does `LIB_NAMESPACE` need to be `camelize` as well? Should this be `#{(ENV['LIB_NAMESPACE'] || ENV['LIB_NAME']).camelize}`

Reply:
@ericsaupe I don't think so, because `LIB_NAMESPACE` is meant to be the Solidus extension gem namespace, it isn't used as a file path, so it doesn't make sense to pass it in a file path form; I meant to use `LIB_NAMESPACE` in order to skip transformations, so f.e. in this way:

```
LIB_NAME=solidus_graphql_api LIB_NAMESPACE=SolidusGraphQLAPI bundle exec rake test_app
```

Do you think we could refactor in a better way the flow? I was thinking about that but unfortunately it didn't come me to mind anything better

Reply:
Makes sense to me! Thanks for the explanation.